x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/inference/loadingservice/ModelLoadingService.java

```diff
@@ -353,9 +353,9 @@ private void auditIfNecessary(String modelId, MessageSupplier msg) {
             logger.trace(() -> new ParameterizedMessage("[{}] {}", modelId, msg.get().getFormattedMessage()));
             return;
         }
-        auditor.warning(modelId, msg.get().getFormattedMessage());
+        auditor.info(modelId, msg.get().getFormattedMessage());
```

Comment:
A model being evicted from the cache is a routine event and not a warning level event. I would prefer this to be debug as it really is a non-issue (the model will be reloaded if required) but there is very little insight into the internals of the cache (which models are loaded etc). Once we have a model management system this should be debug or trace.

I'm raising here for discussion before I forget but I'd be happy to revert the change for this PR